akka-docs/src/main/paradox/project/scala3.md

```diff
@@ -0,0 +1,21 @@
+# Scala 3 support
+
+Akka has experimental support for Scala 3.
+
+## Using 2.13 artifacts in Scala 3
+
+You can use [CrossVersion.for3Use2\_13](https://scala-lang.org/blog/2021/04/08/scala-3-in-sbt.html#using-scala-213-libraries-in-scala-3)
```

Comment:
that `\` is needed?

Reply:
Looks like it's not.

(vim highlighted it as an error, but that might be a bug in vim's syntax highlighting - it seems to work fine with or without the escaping)

I'll simplify